.gitignore

```diff
@@ -1,5 +1,3 @@
-.idea
 .pub
-packages
 .packages
-build
\ No newline at end of file
+pubspec.lock
```

Comment:
I do check in the lockfile. It doesn't affect consumers of dart_style, but it helps ensure everyone hacking on it gets the same experience.